collection-scripts/gather_sos_networking

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+BASE_COLLECTION_PATH="/must-gather"
+LOG_PATH="${BASE_COLLECTION_PATH}/sos_info/networking"
+
+# use quorum-guard becuase it is privileged and uses hostNetwork
+QUORUM_GUARD_DATA=($(oc get pods -n openshift-etcd -l name=etcd-quorum-guard -o jsonpath='{range .items[*]}{.metadata.name}{","}{.status.hostIP}{" "}{end}')) || {
+  echo "ERROR: No networking data was collected"
+  exit 0
+}
+
+if [[ -z $QUORUM_GUARD_DATA ]];then
+  echo "ERROR: No data was returned from oc request"
+  exit 0
+fi
+
+set -o nounset
+
+COMMANDS=(
+  ip_-4_rule
+  ip_-6_rule
+  ip_maddr_show
+  ip_neigh_show_nud_noarp
+  ip_-o_addr
+  ip_-s_-d_link
+  ip_-6_route_show_table_all
+  ip_-d_address
+  ip_mroute_show
+  ip_netns
+  ip_route_show_table_all
+  ip_-s_-s_neigh_show
+)
+
```

Comment:
The script can be more readable if the commands are with spaces and the output file is replaced with underscores. The following works for me:
```
COMMANDS=(
  "ip -4 rule"
  "ip -6 rule"
  "ip maddr show"
  "ip neigh show nud noarp"
  "ip -o addr"
  "ip -s -d link"
  "ip -6 route show table all"
  "ip -d address"
  "ip mroute show"
  "ip netns"
  "ip route show table all"
  "ip -s -s neigh show"
)
```
and 
```
oc rsh  -n openshift-etcd $pod $command > ${node}-${command// /_}.log || true
```

Reply:
makes sense I just created array with filenames but yours is easier to read